src/Components/Components/src/EventCallback.cs

```diff
@@ -55,13 +55,13 @@ public Task InvokeAsync(object arg)
         {
             if (Receiver == null)
             {
-                return EventCallbackWorkItem.InvokeAsync<object>(Delegate, arg);
+                return EventCallbackWorkItem.InvokeAsync<object>(Delegate!, arg);
```

Comment:
How do we know `Delegate` is nonnull here?

Reply:
+1, it can be weird, but you could pass in an empty eventcallback since its a struct and can be default initialized. Maybe we should throw here? 

This is also another place where changing the code adds value. If we simply trick the compiler using `!` we obscure an existing condition (a nullref) where we can be providing a more meaningful error, like this EventCallback was not initialized.

Reply:
We don't, but `EventCallbackWorkItem` allows a null valued delegate. Removing it is probably the right thing to do.